tests/Microsoft.Identity.Web.Test/AzureADB2COpenIDConnectEventHandlersTests.cs

```diff
@@ -0,0 +1,107 @@
+﻿// Copyright (c) Microsoft Corporation. All rights reserved.
+// Licensed under the MIT License.
+
+using Microsoft.AspNetCore.Authentication;
+using Microsoft.AspNetCore.Authentication.OpenIdConnect;
+using Microsoft.AspNetCore.Http;
+using Microsoft.Identity.Web.Test.Common.TestHelpers;
+using Microsoft.IdentityModel.Protocols.OpenIdConnect;
+using NSubstitute;
+using Xunit;
+
+namespace Microsoft.Identity.Web.Test
+{
+    public class AzureADB2COpenIDConnectEventHandlersTests
+    {
+        private const string PathBase = "/PathBase";
+        private const string DefaultUserFlow = "DefaultUserFlow";
+        private const string NonDefaultUserFlow = "NonDefaultUserFlow";
+        private string _defaultIssuer = $"IssuerAddress/{DefaultUserFlow}/";
+        private string _nonDefaultIssuer = $"IssuerAddress/{NonDefaultUserFlow}/";
```

Comment:
not sure what you mean by default and non-default issuer and how that's impacted by the user flow? the user flow is independent of the issuer. maybe we can sync later, i think confused. :)

Reply:
If the user flow in the `RedirectContext` is different from the default user flow (which is susi), then `IssuerAddress` is updated to replace the default user flow with the one from context.

https://github.com/AzureAD/microsoft-identity-web/blob/41798cebfe28a75f2984beca8407018fc9e77b72/src/Microsoft.Identity.Web/AzureADB2COpenIDConnectEventHandlers.cs#L28-L53